Update PoS overview to add a more on-point lede (#6651)
ethereum/ethereum-org-website: src/content/developers/docs/consensus-mechanisms/pos/index.md

sidebar: true
---

> Proof-of-stake (PoS) is a a [consensus mechanism](/developers/docs/consensus-mechanisms/) that Ethereum is moving to, away from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/), because PoS is thought to be more secure, less energy-intensive, and better for implementing new scaling solutions. While it has always been the plan to transition to PoS, proof-of-stake is also more complex than proof-of-work, and refining the mechanism has taken years of research and development. The challenge now is to implement PoS on the live Ethereum network. This process is called ["The Merge"](/upgrades/merge/).

Reviewer: We always spell out proof-of-stake/work fully [as per our style guide.](https://ethereum.org/en/contributing/style-guide/#proof-of)

> Proof-of-stake (PoS) is a a [consensus mechanism](/developers/docs/consensus-mechanisms/) that Ethereum is moving to, away from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/), because PoS is thought to be more secure, less energy-intensive, and better for implementing new scaling solutions. While it has always been the plan to transition to PoS, proof-of-stake is also more complex than proof-of-work, and refining the mechanism has taken years of research and development. The challenge now is to implement PoS on the live Ethereum network. This process is called ["The Merge"](/upgrades/merge/).

Reviewer: ```suggestion
Proof-of-stake (PoS) is the [consensus mechanism](/developers/docs/consensus-mechanisms/) that Ethereum will use after [The Merge](/upgrades/merge). Ethereum is moving off of [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/) to proof-of-stake because it is more secure, less energy-intensive, and better for implementing new scaling solutions. While it has always been the plan to transition to proof-of-stake, it is also more complex than proof-of-work, and refining the mechanism has taken years of research and development. The challenge now is to implement proof-of-stake on Ethereum Mainnet. This process is called ["The Merge"](/upgrades/merge/).
```
